lib/ClangImporter/SwiftDeclSynthesizer.cpp

```diff
@@ -1935,6 +1939,10 @@ SwiftDeclSynthesizer::makeOperator(FuncDecl *operatorMethod,
   topLevelStaticFuncDecl->setBodySynthesizer(synthesizeOperatorMethodBody,
                                              operatorMethod);
 
+  if (opKind == clang::OverloadedOperatorKind::OO_Exclaim) {
```

Comment:
Is there a unary op check we can use instead of individual ifs to make this work for all prefix ops?

Or if not just create a static set of prefix ops and then check if opKind is in that set?

Reply:
Hmm, for some reason I assumed that C++ has postfix unary arithmetic operators, but that's not the case.
You're right, let's check for the number of params to see if the operator is a prefix operator.